index.html

```diff
@@ -44,4 +44,171 @@ <h1 class="headline-heading heading-large">I'm Mariane <br> Glad to see you!</h1
+          </p>
+          <ul class="works-card-tags">
+            <li class="works-card-tag">html</li>
+            <li class="works-card-tag">css</li>
+            <li class="works-card-tag">javascript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </li>
+    </ul>
+  </section>
+  <!-- Works Section Ends Here -->
+
+  <!-- About Section starts here -->
+
+  <section id="about" class="about">
+    <div class="headline-skill-container">
+      <div class="headline">
+        <h1 class="headline-heading heading-large">About <br> Myself</h1>
+        <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product ,
+          feature
+          or website
+          Look
+          through some of my work
+          and experience! If you like what you see and have a project you need coded, don’t hestiate to
+          contact
+          me.
+        </p>
+        <p class="headline-sm-suport">LET’S CONNECT</p>
+        <ul class="social-icons">
+          <li><a href="#"><img src="./Img/-twitter.png"></i></a></li>
+          <li><a href="#"><img src="./Img/Linkedin icon.png"><i class="fa-brands fa-linkedin"></i></a></li>
+          <li><a href="#"><img src="./Img/-medium.png"><i class="fa-brands fa-medium"></i></a></li>
+          <li><a href="#"><img src="./Img/github.png"><i class="fa-brands fa-github"></i></a></li>
+          <li><a href="#"><img src="./Img/victory-icon.svg"><i class="fa-brands fa-facebook"></i></a></li>
+        </ul>
+        <button>Get my resume</button>
+      </div>
+      <ul class="about-skills-container">
+        <li class="about-skill-container">
+          <div class="about-skill-heading">
+            <h4 class="about-heading-small">Languages</h4>
+            <i class="fa-solid fa-angle-down"></i>
+          </div>
+          <div class="about-skill-body-container">
+            <div class="about-skill-body">
+              <img class="about-skill-body-img" src="./Img/icon-js.png" alt="js">
+              <h5 class="about-skill-text">Javascript</h5>
+            </div>
+            <div class="about-skill-body">
+              <img class="about-skill-body-img" src="./Img/icon-html.png" alt="html">
+              <h5 class="about-skill-text">HTML</h5>
+            </div>
+            <div class="about-skill-body">
+              <img class="about-skill-body-img" src="./Img/icon-css.png" alt="css">
+              <h5 class="about-skill-text">CSS</h5>
+            </div>
+          </div>
+        </li>
+        <li class="about-skill-container">
+          <div class="about-skill-heading">
+            <h4 class="about-heading-small">Frameworks</h4>
+            <i class="fa-solid fa-angle-right"></i>
+          </div>
+        </li>
+        <hr>
+        <li class="about-skill-container">
+          <div class="about-skill-heading">
+            <h4 class="about-heading-small">Skills</h4>
+            <i class="fa-solid fa-angle-right"></i>
```

Comment:
- [ ] You have used fontawsome `icon` with `<i>` tag but haven't linked the `fontawsome` in `<head>` tag of the HTML file. Thats why `<i>` tag is not working properly. Kindly fix that issue.
- You can consider follow this [article](https://www.w3schools.com/icons/fontawesome_icons_intro.asp).
- Or you can copy the images given in the template and use it using `img` tag, you don't need fontawesome then.
- ![image](https://user-images.githubusercontent.com/60083437/235313493-b9ff99bd-356b-4316-a701-dd8ee3d96299.png)